src/components/FormsUI/Select/Select.tsx

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import { FieldProps } from 'formik';
+import Select from 'components/Select';
+
+interface OptionBase {
+  label: string;
+  value: string;
+}
+
+interface SelectWrapperProps<Option extends OptionBase> extends FieldProps {
+  options: Option[];
+}
+
+export function SelectWrapper<Option extends OptionBase>({
```

Comment:
Let's name it and its folder `SelectField`. Also put the folder component in `components`